spec/javascripts/components/story/story_description_icon_spec.js

```diff
@@ -0,0 +1,29 @@
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
+import StoryDescriptionIcon from 'components/story/StoryDescriptionIcon';
+
+beforeEach(() => {
+  jasmineEnzyme();
+});
+
+describe('<StoryDescriptionIcon />', () => {
+  describe('When description is null',() => {
+    it('renders nothing', () => {
+      const props = {
+        description: null
+      };
+      const wrapper = shallow(<StoryDescriptionIcon {...props} />);
+      expect(wrapper.find(StoryDescriptionIcon)).not.toExist();
```

Comment:
It would be better to use this matcher here https://github.com/FormidableLabs/enzyme-matchers/tree/master/packages/jasmine-enzyme#tobeemptyrender